etl/steps/data/garden/bp/2022-07-11/shared.py

```diff
@@ -178,16 +178,10 @@ def load_income_groups() -> pd.DataFrame:
         Income groups data.
 
     """
-    income_groups = (
-        catalog.find(
-            table="wb_income_group",
-            dataset="wb_income",
-            namespace="wb",
-            channels=["garden"],
-        )
-        .load()
-        .reset_index()
-    )
+    # Load the latest version of the WorldBank dataset for income grups.
+    latest_wb_dataset_path = sorted((DATA_DIR / "garden/wb/").glob("*/wb_income"))[-1]
```

Comment:
Agree. Using `PathFinder` would be a bit tricky here since we're in `shared.py` and can't just call `PathFinder(__file__)`. It'd be perhaps better to pass the WB dataset into `load_income_groups` from the dataset module, though I ended up with the hardcoded version for simplicity.